wagtailsharing/tests/test_urls.py

```diff
@@ -2,9 +2,10 @@
 
 from django.test import TestCase
 
-from mock import patch
 from wagtail.core import urls as wagtail_core_urls
 
+from mock import patch
```

Comment:
Could this actually be imported from `unittest.mock`? Then we could remove the dependency on the external `mock` package.